src/tab.panel.ts

```diff
@@ -56,10 +62,11 @@ export default class TabPanel extends LitElement {
   @property({ reflect: true })
   readonly version: string = packageJson.version;
 
-  protected override firstUpdated() {
-    this.setAttribute('role', 'tabpanel');
-    this.id = this.#id;
```

Comment:
> Tab Panel and Tab are now accessibly labeled and associated with each other when they're added to Tab Group after initial render.

I would have left these alone given the upcoming rework. Changing them, however, let me fix the bug above. I fixed it by calling [`#setAriaAttributes()`](https://github.com/CrowdStrike/glide-core/pull/895/files#diff-cfa9ddb827a45a55c1fb21f5a32268f138d6cf89ca688cc9e0cd0c443d33dafaR339) in `#onDefaultSlotChange()`. But then `relatedPanel?.id` is falsy at that point because Tab Panel's `firstUpdated()` has yet been called, and `firstUpdated()` was where Tab Panel was being given an `id`.



https://github.com/CrowdStrike/glide-core/pull/895/files#diff-cfa9ddb827a45a55c1fb21f5a32268f138d6cf89ca688cc9e0cd0c443d33dafaR274-R275